### ISSUE TITLE
Add react-native courses

### ISSUE THREAD

**Issue by ishandeveloper** [ishandeveloper/Coursify-hacktoberfest#4]
### Follow the guidelines in 'How To Contribute' section of README.md
and add a course that you've taken before or would like to recommend to others, related to react-native.

**Comment by alphaCoder-69** [comment]
Can I work on it?

**Comment by ishandeveloper** [comment]
Yes @alphaCoder-69 you can. There's no need for assigning an issue, you can go ahead and make a PR ✨
